### ISSUE TITLE
Improve accuracy

### ISSUE THREAD

**Issue by sepidehamiri** [amiralansary/rl-medical#26]
Hi,

First of all, thank you so much for your great and useful code. I'm trying to apply this code to my dataset, 37 pancreas ducts (small size organ) and I have 2 landmarks. training error is around 0.5, but test error is around 16-20. I tried with the default parameters and sum change on, step_per_episode, batch size, target update freq, and learning rate. Do you have any suggestions for decreasing the gap between these two errors?

Thanks in advance
